README.md

```diff
@@ -2,7 +2,10 @@
 
 ## Installation
 
-Copy the extension to `phpBB/ext/phpbb/skeleton`
+To install the source code from this repository, clone this extension repository to `phpBB/ext/phpbb/skeleton`:
+
+	$ git clone https://github.com/phpbb-extensions/phpbb-ext-skeleton.git
```

Comment:
This is actually meant for people who already know what they are doing. Once all these PRs are done, will be have releases, and that will be the recommended install instruction to use. However, we have no releases so we can't mention that here yet.